_articles/how-to-contribute.md

```diff
@@ -31,6 +31,10 @@ Contributing to open source can be a rewarding way to learn, teach, and build ex
 
 Why do people contribute to open source? Plenty of reasons!
 
+### top techs joined open source
```

Comment:
Remember to capitalize the first letter when starting a paragraph. I would change this title to  "Top tech companies that joined open source projects" or something like that.

Reply:
You could also add the word have in there so that it states “Top tech companies that have joined open source projects” if that isn’t to much writing in order to still grab attention. I like the idea of Vinay brought up about bringing that up though since I would read it and good catch on the capitalization because I was thinking the same thing and also about making sure to capitalize company names like Microsoft.